### PR TITLE
[INLONG-6363][Dashboard] InLong Group supports Kafka MQ

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes https://github.com/apache/inlong/issues/6363

- The details are as follows:

image

### CODE REVIEW

File: inlong-dashboard/src/metas/groups/defaults/Kafka.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    rules: [{ required: true }],
+    initialValue: 1,
+    suffix: i18n.t('meta.Group.PartitionUnit'),
+    props: {
+      min: 1,
+      precision: 0,
+    },
+  })
+  @I18n('meta.Group.AccessSize')
+  replicationFactor: number;
```
Comment (healchow) [inline inlong-dashboard/src/metas/groups/defaults/Kafka.ts:50]: The `numPartitions` and `replicationFactor` should be in InlongStream layer.

Comment (bluewang) [reply inlong-dashboard/src/metas/groups/defaults/Kafka.ts]: ok. I changed it now

Comment (bluewang) [reply inlong-dashboard/src/metas/groups/defaults/Kafka.ts]: Currently, all Kafka topics in an inlong group must have the same number of partitions and replication factors.

File: inlong-dashboard/src/locales/en.json [hunk 2]
```diff
@@ -233,6 +233,9 @@
   "meta.Group.AccessPeakPerSecond": "PeakRecords",
   "meta.Group.Stripe/Second": "Stripe / Second",
   "meta.Group.SingleStripMaximumLength": "Single Max Length",
+  "meta.Group.Partition": "Number of partitions",
+  "meta.Group.PartitionUnit": "Indivual",
```
Comment (healchow) [inline inlong-dashboard/src/locales/en.json:237]: It is not necessary to use a `unit`.

File: inlong-dashboard/src/locales/cn.json [hunk 3]
```diff
@@ -233,6 +233,8 @@
   "meta.Group.AccessPeakPerSecond": "每秒接入峰值",
   "meta.Group.Stripe/Second": "条/秒",
   "meta.Group.SingleStripMaximumLength": "单条最大长度",
+  "meta.Group.Partition": "分区数",
```
Comment (leezng) [inline inlong-dashboard/src/locales/cn.json:236]: The key use `meta.Group.Kafka.Partition`. Others similar to the same need to be modified.

**Review by healchow — APPROVED** [review]

**Review by haifxu — APPROVED** [review]

**Pull request merged**